### DISCUSSION TITLE
Duda kata - modulo 2 / Instalación python-dateutil

### DISCUSSION THREAD

**Discussion by andresmm87** [LaunchX-InnovaccionVirtual/CursoIntroPython#24] [Q&A] [answered]
Hola que tal, tuve un detalle al intentar instalar desde consola el paquete _dateutil_, después de insgresar el comando tal y como viene en la documentación de la kata, python me regresaba un mensaje que la versión de Python no era la necesaria para poder instalar el paquete (según entendí), sin embargo, para un bootcamp en el que estoy de ciencia de datos uso la libreria IPython y al parecer se instaló correctamente, habría algún detalle si continuo las katas de esta forma?

![image](https://user-images.githubusercontent.com/33568700/153082496-995398df-61a5-40a4-a0a3-530fab99951a.png)

**Comment by FernandaOchoa** [comment, marked as answer]
Hola, revisando tu captura, estás ejecutando toda la kata sobre dos entornos. Te recomiendo desactivarlos y trabajar con uno nada más para que no tengas inconvenientes. 
Justamente como lo menciono en la documentación puede ser que tengas otros proyectos y no queremos que se vean afectados, por lo que deben tener un entorno únicamente para este ciclo.

Algo que también puedo notar es la ambigüedad de la captura, ya que no se ve desde el inicio el error de la instalación del comando. 

Dejo la kata completa resuelta, aunque como mencioné de nada les sirve tener las respuestas si no logran construir su proceso de aprendizaje. De este modo solo están aprendiendo por repetición y este método lo único que logra es dependencia de la muestra de la que aprenden, el día que quitan esa muestra, el conocimiento se va con ella.

✨ Ánimo, tratemos de salir de la zona de confort ✨ 

![image](https://user-images.githubusercontent.com/9124597/153083838-9f4ecbe6-e663-4771-b62d-751bc798f8cc.png)
